akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -1514,7 +1514,6 @@ final class Flow[In, Out, Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Gr
    *
    * '''Cancels when''' downstream cancels
    */
-  @deprecated("Use recoverWithRetries instead.", "2.4.4")
```

Comment:
Why doesn't this apply anymore? (I guess it should have been remove on the way, but it is still deprecated, right.)

Reply:
Ah, this was incorrectly deprecated, so I fixed #25530 as part of this PR